### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by Josh3845**

### CODE REVIEW

File: js-intro.js [hunk 1]
```diff
@@ -23,18 +24,40 @@ var myCat = "Garfield"
...
+        return dog
+    }else {
+        return cat
+    }
+}
 
+console.log(length(myDog, myCat));
+// 2b. Create a function that takes in two arguments and returns an array containing the two variables. Expected output: ["Snoopy", "Garfield"]
+let pet = [myDog,myCat]
+console.log(pet)
```
Comment (DanteCypress) [inline js-intro.js]: Hey Josh, logic is sound. Let's put this into a function now. We can do this by:
ES5: `function intoArr(str,str2){
}`
ES6: `const intoArr = (str,str2) =>{
}`

File: js-intro.js [hunk 2]
```diff
@@ -23,18 +24,40 @@ var myCat = "Garfield"
...
+// function lower (dog1,cat1) {
+//     if(dog1 === "Snoopy"){
+//     return dog1.tolowerCase()
+// }else if (cat1 === "Garfield"){
+//     return cat1.tolowerCase()
+// }else{
+//     return "hi"
+// }
+// }
+// console.log(lower(myDog,myCat))
```
Comment (DanteCypress) [inline js-intro.js]: Hey Josh,
Let's think about why we're using an if statement here? What if dog1 wasn't Snoopy, then it wouldn't return `dog1.toLowerCase()`. Consider removing the `if` statement or beefing up the `if` statement to look at the case sensitive cases.

File: js-intro.js [hunk 3]
```diff
@@ -46,7 +69,20 @@ var myMessage = "Learning is fun!"
...
+let yup = testString.split("")
+
+function remo (sep){
+  for( let i =0 ; i < yup.length; i++)
+  const win = yup.length[i]
+  if(yup[i] === "a" || yup[i] === "e" || yup[i] ==="i" || yup[i] ===
+"o" ||yup[i] === "u"){
+return yup[i].join("")
+}
+}
```
Comment (DanteCypress) [inline js-intro.js:102]: Hey Josh, you're on to something here. Consider the following:
What do you want to do with the values in your array, before you join the array back together? Below I have added a template for a function, you just need to fill it in with the logic
```
const noVowels = (str) =>{
 let yup = str.split('')

 for(let i =0; i<yup.length;i++){
     if(){

     }
 }
}
```

File: js-intro.js [hunk 4]
```diff
@@ -56,7 +92,28 @@ var testString = "thisisateststring"
...
+  const {animal} = toonimals
+  for (let i = 0; i < toonimals.length; i++)
+var array = {}
+  if(animal === "cat"){
+    return array
+  }else{
+    return "hi"
+  }
+}
+console.log(cats(toonimals));
```
Comment (DanteCypress) [inline js-intro.js]: Hey Josh, Consider the following:
```
let ifCat = (listOfToonimals) => {
     let arr = []
     listOfToonimals.forEach(function (toonimal){
         if(toonimal.animal=="cat"){
             arr.push(toonimal)
         }
     })
  
     return arr
 }
console.log(ifCat(toonimals));
```

File: js-intro.js [hunk 5]
```diff
@@ -56,7 +92,28 @@ var testString = "thisisateststring"
...
+cats =(...toonimals) => {
+  const {animal} = toonimals
+  for (let i = 0; i < toonimals.length; i++)
+  if(animal!=== "cat"){
+    return array
+  }else{
+    return "hi"
+  }
+}
+console.log(cats(toonimals));
```
Comment (DanteCypress) [inline js-intro.js:132]: Hey Josh, look above and see if you can solve this problem. Make sure to look up any methods or high-order function you need clarification on

**Review by DanteCypress — CHANGES_REQUESTED** [review]

**Review by sjproctor — APPROVED** [review]
Nice work! Those changes look good. Go ahead and merge your code.

**Pull request merged**